packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -215,62 +214,11 @@ contract OptimismPortal_Test is Portal_Initializer {
         assertEq(address(op).balance, NON_ZERO_VALUE);
     }
 
-    function test_cannotVerifyRecentWithdrawal() external {
```

Comment:
Not removed, this was just relocated into the new test contract and renamed to `test_finalizeWithdrawalTransaction_revertsOnRecentWithdrawal()`.